### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by MD-IS**

### CODE REVIEW

**Comment by falzonv** [comment]
Hello,

Thank you very much for the initiative and the work done!

Except for a small remark (I will create the related comment after this message), everything looks good :-)

Best regards.

File: app/src/main/res/values-in/strings.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="search_app_hint">Cari aplikasi</string>
+
+    <!-- About dialog -->
+    <string name="app_name">Discreet Launcher</string>
+    <string name="author">Vincent Falzon</string>
+    <string name="licence_name">GNU General Public License v3.0</string>
+    <string name="license_link">https://gnu.org/licenses/gpl-3.0.txt</string>
+    <string name="website_description">Saran peningkatan dan lapor isu di</string>
+    <string name="website_link">https://vincent-falzon.com</string>
+    <string name="translation_credit">Indonesian translation by Dawud Irfan</string>
```
Comment (falzonv) [inline app/src/main/res/values-in/strings.xml:47]: The string "translation_credit" should also be translated in Indonesian

Comment (MD-IS) [reply app/src/main/res/values-in/strings.xml]: done

**Comment by falzonv** [comment]
Great, thank you for the fix!

I am currently preparing the next release and a couple of new strings will be added.
Let's wait a bit before merging the PR, so we can update directly here and merge once at the end just before the release.

File: app/src/main/res/values-in/strings_settings.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="set_immersive_mode_help">Tidak ada batang sistem. (Bisa dipaksakan dengan tekan lama di layar rumah.)</string>
+    <string name="set_reverse_interface">Balikkan Antarmuka</string>
+    <string name="set_reverse_interface_help">Laci Aplikasi di atas, dan favorit di bawah layar rumah</string>
+    <string name="set_hide_menu_button">Sembunyikan tombol menu</string>
+    <string name="set_hide_menu_button_help">Selalu bisa diakses dari ikon aplikasi peluncur</string>
+    <string name="set_disable_app_drawer">Nonaktifkan Laci Aplikasi</string>
+    <string name="set_swipe_towards_left">Geser ke kiri</string>
+    <string name="set_swipe_towards_right">Geser ke kanan</string>
+    <string name="set_swipe_towards_leftright_no_action">Tidak ada</string>
+
```
Comment (falzonv) [inline app/src/main/res/values-in/strings_settings.xml:139]:
```suggestion
```

**Comment by falzonv** [comment]
Perfect, let's merge then!
For future updates, please check ticket #80 :-)

**Comment by falzonv** [comment]
@MD-IS

I just noticed a possible misunderstanding on the 3 strings in "Sections titles" of this commit:
https://github.com/falzonv/discreet-launcher/commit/f8560ec84c9bb121a05b00c88d64b8d86583ead9#diff-57bd8fe85c500ed06ccdeef5916b0db01d78d19d3dedf4ba07c8f9fb7728cef9

```
--- <string name="appearance_settings_title_general">General appearance</string>
--- <string name="appearance_settings_title_applications">Appearance of apps</string>
+++ <string name="appearance_settings_title_general">General</string>
+++ <string name="appearance_settings_title_clock">Clock</string>
+++ <string name="appearance_settings_title_applications">Apps</string>
```

Two actions were done:
- Simplify the existing titles (from "General appearance" to "General", and from "Appearance of apps" to "Apps") because the text "Appearance" is already displayed as the title of the window
- Add a new "Clock" title

**Could you confirm that this is correct also in your translation?** (the commit https://github.com/falzonv/discreet-launcher/pull/123/commits/dbd1e890589b52c3707bb432b4ee3b30274dde51 only adds the Clock title)

=> If not, no need to create a new PR, just write the correct strings here and I will copy-paste them in the file :-)

**Comment by MD-IS** [comment]
<string name="appearance_settings_title_general">Umum</string>
    <string name="appearance_settings_title_clock">Jam</string>
    <string name="appearance_settings_title_applications">Aplikasi</string>

**Comment by falzonv** [comment]
Perfect, integrated!

**Pull request merged**